cvat/apps/engine/media_extractors.py

```diff
@@ -868,16 +882,17 @@ def _compress_image(source_image: av.VideoFrame | io.IOBase | Image.Image, quali
             image = Image.fromarray(image, mode="L") # 'L' := Unsigned Integer 8, Grayscale
             image = ImageOps.equalize(image)         # The Images need equalization. High resolution with 16-bit but only small range that actually contains information
 
-        converted_image = image.convert('RGB')
+        if image.mode != 'RGB':
```

Comment:
Perhaps we could save even more memory by not converting images of mode `L`? If an image is grayscale, we could just encode it as-is.

Reply:
done